src/graph/entities/qg_edge.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include "edge.h"
+#include "qg_node.h"
+#include <string.h>
+
+struct QGEdge {
+    const char *alias;
+    const char **reltypes;
+    int *reltypeIDs; // TODO can be uint save for GRAPH_NO_RELATION
+    QGNode *src;
+    QGNode *dest;
+    uint minHops;   /* Minimum number of hops this edge represents. */
+    uint maxHops;   /* Maximum number of hops this edge represents. */
+    uint id;
+};
+
+typedef struct QGEdge QGEdge;
+
+/* Creates a new edge, connecting src to dest node. */
+QGEdge* QGEdge_New(QGNode *src, QGNode *dest, const char *relationship, const char *alias);
+
+QGEdge* QGEdge_Clone(const QGEdge *orig);
```

Comment:
Please document function

Reply:
Sorry, a ton of functions are missing documentation still! Once all the interfaces are finalized, I'll be sure to document all of them.